pl_examples/basic_examples/conf/config.yaml

```diff
@@ -0,0 +1,11 @@
+defaults:
+  - models: basic
```

Comment:
use model for the group name, not models.
this way when you override from the command line it looks like:

```
python cpu_template.py model=resnet50
```